UKHomeOffice/sgmr-service
# Jest testing suite

## Description
This branch provides jest testing (with enzyme and testing-library) for core sgmr-service components. As well as this, the git hooks package [Husky](https://www.npmjs.com/package/husky) has been added to provide git hook functionality. Thus far, it has only been added for `git commit`.

## How to test
1. Pull the branch
2. Run `npm install`
3. Run `npm run test:coverage` 
4. You should see 101 tests passed spanning across 33 testing suites

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
